### PR TITLE
[BUILD] Fix Import Abseil-cpp

### PR DESCRIPTION
**Pull request by LeiZhang-Hunter**
…ink you can make a judgment when introducing abseil-cpp. If abseil is not introduced, then use find_package

Fixes # (issue)

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Unit tests have been added
* [ ] Changes in public API reviewed

### CODE REVIEW

**Review by lalitb — APPROVED** [review]
LGTM. CI is failing for formatting, so need to be fixed before merging.

File: CMakeLists.txt [hunk 1]
```diff
@@ -377,7 +377,10 @@ if(WITH_PROMETHEUS)
 endif()
 
 if(WITH_ABSEIL)
-  find_package(absl CONFIG REQUIRED)
+  if(TARGET absl::strings)
```
Comment (ThomsonTan) [inline CMakeLists.txt:380]: nit: put a NOT to negate the condition in `if()` so the empty if block can be avoided?

Comment (LeiZhang-Hunter) [reply CMakeLists.txt]: I have modified it.

**Review by marcalff — APPROVED** [review]
LGTM, thanks for the fix.

**Pull request merged**